requirements-dev.txt

```diff
@@ -1,4 +1,7 @@
 -r requirements.txt
+aiohttp==3.7.4
+asyncio==3.4.3
+asynctest==0.13.0
```

Comment:
I think new dependencies should be included here: https://github.com/opencybersecurityalliance/stix-shifter/blob/develop/stix_shifter/requirements.txt

Reply:
done